src/python/examples/inference_example.py

```diff
@@ -0,0 +1,45 @@
+import pathlib
+
+import numpy as np
+import tritonclient.http as httpclient
+import tritonserver
+from tritonfrontend import KServeHttp
+
+model_path = f"{pathlib.Path(__file__).parent.resolve()}/../test/test_model_repository"
+server_options = tritonserver.Options(
+    server_id="ExampleServer",
+    model_repository=model_path,
+    log_error=True,
+    log_warn=True,
+    log_info=True,
+)
+server = tritonserver.Server(server_options).start()
```

Comment:
Probably want to use `start(wait_until_ready=True)` to avoid client sending request before model is ready or something